client/my-sites/site-settings/form-seo.jsx

```diff
@@ -224,7 +227,11 @@ export const SeoForm = React.createClass( {
 		const isDisabled = isSitePrivate || isSubmittingForm || isFetchingSettings;
 		const isSaveDisabled = isDisabled || isSubmittingForm || ( ! showPasteError && invalidCodes.length > 0 );
 
-		const sitemapUrl = `https://${ slug }/sitemap.xml`;
+		const seoTitle = siteDescription.length
+			? `${ siteTitle } | ${ siteDescription }`
```

Comment:
this is something we are aware about and not yet addressing. there's concurrent work going on to give the user the control to set how this will display regardless of theme. I don't think we will get to that in this PR, but thanks for raising that up